lib/libc/gen/mips/setjmp.S

```diff
@@ -66,11 +67,16 @@ NON_LEAF(setjmp, SETJMP_FRAME_SIZE, ra)
 	PTR_ADDU	a1, sp, CALLFRAME_SIZ		# pointer to stack_t
 	jal		_C_LABEL(sigaltstack)
 
-	move		a0, s0				# restore jmpbuf
+    move        a0, s0
```

Comment:
Please configure your `vim` correctly for assembly files. Indentation should be 8 spaces.

Reply:
done